runbotics/runbotics-desktop/src/action/google/google.action-handler.ts

```diff
@@ -187,4 +188,17 @@ export default class GoogleActionHandler extends StatelessActionHandler {
             values: [],
         };
     }
+    private inputArrayParser(values: unknown[][] | string): unknown[][] {
```

Comment:
in the schema you specified values can be: string[] | (string | number)[][] so you should create a type for that